packages/typespec-go/test/armdatabasewatcher/zz_models.go

```diff
@@ -57,6 +54,42 @@ type Datastore struct {
 	KustoClusterDisplayName *string
 }
 
+// ErrorAdditionalInfo - The resource management error additional info.
```

Comment:
The structures appear to be the same in all the tests. Is that by chance or are these types that could be shared from the armCore library?

Reply:
I believe this, and other types, come from `@azure-tools/typespec-azure-resource-manager`.